ivy_tests/test_ivy/test_frontends/test_numpy/test_matrix/test_methods.py

```diff
@@ -216,7 +264,9 @@ def test_numpy_matrix_any(
         method_input_dtypes=[],
         method_all_as_kwargs_np={
             "axis": axis,
+            "dtype": dtype[0],
         },
         frontend=frontend,
+        on_device='numpy',
```

Comment:
here should then be `on_device=on_device`

Reply:
Why? It was testing tensorflow back then

Reply:
it's not related to that, `on_device` is related to `cpu`, `gpu`, etc testing

and since this is a numpy frontend, it should be compatible with the various backend so testing against a tensorflow backend shouldn't be an issue (:

Reply:
so the current `on_device="numpy"` is causing tests to fail because it is not a valid value for the particular parameter